server/build.sbt

```diff
@@ -110,6 +115,9 @@ lazy val root = (project in file("."))
     // After 2 transitive steps, do more aggressive invalidation
     // https://github.com/sbt/zinc/issues/911
     incOptions := incOptions.value.withTransitiveStep(2),
+    pipelineStages := Seq(digest, gzip), // plugins to use for assets
```

Comment:
Does this mean that the GzipFilter is no longer necessary?

Reply:
Gzip filter is still needed for dyanmically generated files.  The gzip here is only for the static assets, and actually builds the gzip into the built package, so there's no overhead at run time.